src/components/DeathKill.tsx

```diff
@@ -45,7 +45,10 @@ const DeathKill = ({deathKillTimers, videoHandler}: DeathKillProps) => {
         } else {
             timer = [...splitDeathKill[action]].reverse().find(
                 record => {
-                    return record.startTime < currentTime
+                    // Subtract timeBeforeAction from currentTime because otherwise 
+                    // skipping back would never work as long as the video was playing.
+                    // It would just keep skipping back to the latest event but never 
+                    return record.startTime < currentTime - timeBeforeAction
```

Comment:
i see what u mean hmm im not sure if its an issue though as it is what all players do. but we need to implement so it goes once and then it goes back one